pygmt/src/timestamp.py

```diff
@@ -23,27 +23,39 @@ def timestamp(
     r"""
     Plot the GMT timestamp logo.
 
+    Add the GMT timestamp logo with an optional label at the bottom-left corner
+    of a plot. The timestamp will be in the locale set by the environment
+    variable **TZ** (generally local time but can be changed via
+    ``os.environ["TZ"]``) and its format is controlled by the parameter
+    ``timefmt``. It can also be replaced with any custom text string using the
+    ``text`` parameter.
+
```

Comment:
I am wondering, whether we should mention that there is by default an offset of `("-54p", "-54p")`?
```suggestion
    Add the GMT timestamp logo with an optional label at the bottom-left corner
    of a plot with an ``offset`` of ``("-54p", "-54p")``. The timestamp will be
    in the locale set by the environmentvariable **TZ** (generally local time
    but can be changed via ``os.environ["TZ"]``) and its format is controlled
    by the parameter ``timefmt``. It can also be replaced with any custom
    text string using  the ``text`` parameter.

```